install_gz11_side_by_side.md

```diff
@@ -0,0 +1,70 @@
+# Installing Gazebo11 side by side with new Gazebo
+
+## Supported Gazebo versions and Ubuntu platforms for this tutorial
+
+This tutorial affects the Gazebo Classic users on Ubuntu `Jammy` that plan on
```

Comment:
I would include something like, You can't install Gazebo using packages from Noble and later distributions

Reply:
good idea, [e420960](https://github.com/gazebosim/docs/pull/438/commits/e420960cdd4c362ae2ea04872c6ff7e585ce8805)